jfrog-cli/artifactory/utils/golang/cachevalues.go

```diff
@@ -0,0 +1,53 @@
+package golang
+
+var cache DynamicCache
+
+type DynamicCache struct {
```

Comment:
I suggest renaming to *DependenciesCache*.

Reply:
Done